### DISCUSSION TITLE
[Feature Request] Selecting different layouts

### DISCUSSION THREAD

**Discussion by masterlog80** [linuxserver/Heimdall#1285] [Ideas]
Hello,
Despite [Heimdall](https://github.com/linuxserver/Heimdall) is a very useful application, I believe that adding the possibility to selecting different layouts (e.g. on 3, 4, 5.... columns) would be great.
Also, as mentioned [here](https://github.com/linuxserver/Heimdall/issues/352), [here](https://github.com/linuxserver/Heimdall/issues/1175) and [here](https://github.com/linuxserver/Heimdall/issues/646) having a sort of "grouping" to separate the application by type would definitely improve the usability.

Regards,

[a comment by an automated account is not shown]

**Comment by XobSod** [comment]
+1

[a comment by an automated account is not shown]

**Comment by XobSod** [comment]
+1

[a comment by an automated account is not shown]
